src/metrics/scenario.jl

```diff
@@ -180,3 +180,47 @@ end
 Calculate the mean coral evenness for each scenario for the entire domain.
 """
 scenario_evenness = Metric(_scenario_evenness, (:timesteps, :scenarios))
+
+"""
+    scenario_outcomes(rs::ResultSet, metrics::Vector{Metric})::NamedDimsArray
+
+Get outcomes for a given list of metrics and a result set
+
+# Arguments
+    - `rs` : ResultSet
+    - `metrics` : Vector of scenario Metrics (the ones that start "with scenario_")
+
+# Returns
+    NamedDimsArray with (:timesteps, :scenarios, :outcomes)
```

Comment:
Weird indentation? Is this the auto-formatter again?

Reply:
No, this one is on me D: